molecule/delegated/tests/kubectl/redhat.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from ..util.util import (
+    get_ansible,
+    get_variable,
+    get_from_url,
+    extract_url_from_variable,
+    get_centos_repo_key,
+)
+
+testinfra_runner, testinfra_hosts = get_ansible()
+
+
+def check_ansible_os_family(host):
+    if get_variable(host, "ansible_os_family", True) != "RedHat":
+        pytest.skip("ansible_os_family mismatch")
+
+
+def check_configure_repository(host):
+    if not get_variable(host, "kubectl_configure_repository"):
+        pytest.skip("kubectl_configure_repository is not set")
+
+
+def test_kubectl_gpg_key_present(host):
+    """Check if the GPG key for the kubectl repository is correctly added."""
+    check_ansible_os_family(host)
+    check_configure_repository(host)
+
+    all_keys = host.run(
+        "rpm -qa gpg-pubkey* --qf '%{NAME}-%{VERSION}-%{RELEASE} --> %{SUMMARY}\n%{DESCRIPTION}\n\n'"
+    ).stdout
+    installed_key = get_centos_repo_key(
+        host,
+        "isv:kubernetes OBS Project <isv:kubernetes@build.opensuse.org> public key",
+    )
+    assert installed_key in all_keys
```

Comment:
Redundant check, we just want to know if get_centos_repo_key is the same as in the file